2-ui/3-event-details/1-mouse-events-basics/article.md

```diff
@@ -1,211 +1,212 @@
 
-We usually don't use it for `click` and `contextmenu` events, because the former happens only on left-click, and the latter -- only on right-click. 
+Normalmente no la usamos para eventos `click` y `contextmenu` events, porque sabemos que ocurren solo con click izquierdo y derecho respectivamente. 
 
-From the other hand, `mousedown` and `mouseup` handlers we may need `event.button`, because these events trigger on any button, so `button` allows to distinguish between "right-mousedown" and "left-mousedown".
+Por otro lado, con lo controloadores `mousedown` y `mouseup` vamos a necesitar `event.button` ya que estos eventos se activan con cualquier botón, por lo que `button` nos permitira distinguir entre "mousedown derecho" y "mousedown izquierdo".
 
-The possible values of `event.button` are:
+Los valores posibles para `event.button` son:
 
-| Button state | `event.button` |
+| Estado del botón | `event.button` |
 |--------------|----------------|
-| Left button (primary) | 0 |
-| Middle button (auxillary) | 1 |
-| Right button (secondary) | 2 |
-| X1 button (back) | 3 |
-| X2 button (forward) | 4 |
+| Botón izquierdo (primario) | 0 |
+| Botón central (auxiliar) | 1 |
+| Botón derecho (secundario) | 2 |
+| Botón X1 (atrás) | 3 |
+| Botón X2 (adelante) | 4 |
 
-Most mouse devices only have the left and right buttons, so possible values are `0` or `2`. Touch devices also generate similar events when one taps on them.
+La mayoría de los dispositivos de ratón sólo tienen los botones izquierdo y derecho, por lo que los valores posibles son `0` o `2`. Los dispositivos táctiles también generan eventos similares cuando se toca sobre ellos.
 
-Also there's `event.buttons` property that has all currently pressed buttons as an integer, one bit per button. In practice this property is very rarely used, you can find details at [MDN](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons) if you ever need it.
+También hay una propiedad `event.buttons` que guarda todos los botones presionados actuales en un solo entero, un bit por botón. En la práctica, esta propiedad es raramente utilizada. Puedes encontrar más detalles en [MDN](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons) si alguna vez lo necesitas.
 
-```warn header="The outdated `event.which`"
-Old code may use `event.which` property that's an old non-standard way of getting a button, with possible values:
+```warn header="El obsoleto `event.which`"
+El código puede utilizar la propiedad `event.which` que es una forma antigua no estándar de obtener un botón con los valores posibles:
 
-- `event.which == 1` – left button,
-- `event.which == 2` – middle button,
-- `event.which == 3` – right button.
+- `event.which == 1` – botón izquierdo,
+- `event.which == 2` – botón central,
+- `event.which == 3` – botón derecho.
 
-As of now, `event.which` is deprecated, we shouldn't use it.
+A partir de ahora `event.which` is deprecatedestá en desuso, no deberíamos usarlo.
```

Comment:
as for now  acá traduciría "ahora dejó de".  muy complicado. Mejor:
```suggestion
Ahora `event.which` está en desuso (y calificado como "deprecated"), no deberíamos usarlo.
```

Reply:
```suggestion
Ahora `event.which` está en desuso, no deberíamos usarlo.
```